packages/apidom-parser-adapter-yaml-1-2/test/adapter-browser.ts

```diff
@@ -96,4 +96,34 @@ describe('adapter-browser', function () {
       assert.isTrue(parseResult.isEmpty);
     });
   });
+
+  context('given YAML 1.2 containing a syntax error(1)', function () {
+    specify('return YAML Syntax error as an annotation in parsing result', async function () {
+      const syntaxErrorSpec = `
+        asyncapi: 2.4.0
+        info:
+          version: '1.0.0'
+           title: Something # Badly indented
+      `;
+
+      const parseResult = await adapter.parse(syntaxErrorSpec, { sourceMap: true });
+
+      assert.isTrue(parseResult.errors.toValue()[0] === '(Error YAML syntax error)');
```

Comment:
Let's try to make this assertion a bit more idiomatic.

---

We need to import `toValue` to translate ApiDOM into JavaScript.
```js
import { toValue } from '@swagger-api/apidom-core';
```

```suggestion
      assert.strictEqual(toValue(parseResult.errors.get(0)), '(Error YAML syntax error)');
```